lib/msal-angular/samples/MSALAngularDemoApp/package.json

```diff
@@ -24,7 +24,7 @@
     "rxjs": "^5.1.0",
     "ts-helpers": "^1.1.1",
     "zone.js": "^0.8.4",
-    "@azure/msal-angular": "0.1.2"
+    "@azure/msal-angular": "file:../.."
```

Comment:
Does this help with picking the latest?

Reply:
Yeah, this will install the locally built version of the library, instead of what's published to npm.